programs/launcher/main.cpp

```diff
@@ -987,16 +986,19 @@ launcher_def::launch (eosd_def &instance, string &gts) {
   bf::path pidf  = dd / "eosd.pid";
 
   host_def* host = deploy_config_file (*instance.node);
-
   node_rt_info info;
   info.remote = !host->is_local();
 
   string eosdcmd = "programs/eosd/eosd ";
   if (skip_transaction_signatures) {
     eosdcmd += "--skip-transaction-signatures ";
   }
-  eosdcmd += eosd_extra_args + " ";
-  eosdcmd += "--enable-stale-production true ";
+  if (!eosd_extra_args.empty()) {
+    eosdcmd += eosd_extra_args + " ";
+  }
+  else {
+    eosdcmd += "--enable-stale-production true ";
+  }
```

Comment:
So if extra args are given you don't want the `--enable-stale-production true` ?

Reply:
correct. the user can add --enable-stale-production true along with any other arguments if it is desired, but in cases where a server is being bounced, --enable-stale-production should be replaced with --resync.